addon/helpers/ehook.js

```diff
@@ -4,7 +4,7 @@
  * @see {@link https://github.com/Ticketfly/ember-hook/pull/35}
  */
 import Ember from 'ember'
-const {Helper, deprecate} = Ember
+const {Helper, deprecate, keys} = Ember
```

Comment:
The result of this inquiry applies equally to many other changes in this PR in other files.